include/devenv/seadEnvUtil.h

```diff
@@ -1,13 +1,17 @@
 #pragma once
 
+#include "prim/seadEnum.h"
 #include "prim/seadSafeString.h"
 
 namespace sead
 {
+SEAD_ENUM(RegionLanguageID, JPja, USen, USes, USfr, USpt, EUen, EUes, EUfr, EUde, EUit, EUpt, EUnl, EUru, KRko, CNzh, TWzh)
+
 class EnvUtil
 {
 public:
     static const SafeString& getRomType();
+    static const RegionLanguageID getRegionLanguage();
```

Comment:
```suggestion
    static RegionLanguageID getRegionLanguage();
```
(the const doesn't do anything here)